From 83075f7fc939eccec4358462c70b1998c1e05a61 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Thu, 15 Feb 2024 13:39:52 +0100
Subject: [PATCH 0595/1695] dts: stm32: add wake up parent in pwr_irq node

Refer to EXTI as pwr_irq wake up parent.

Do not apply any configuration to wake up GPIOS by default.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e40753d795f0b6f917d438d6c01ea8b227a2ffe7 dts: stm32: add wake up parent in pwr_irq node]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ia5506944dcdc4ed788bf395439d157defebbd94a
---
 core/arch/arm/dts/stm32mp131.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp131.dtsi b/core/arch/arm/dts/stm32mp131.dtsi
index 4eaeb2904..03dacc6b4 100644
--- a/core/arch/arm/dts/stm32mp131.dtsi
+++ b/core/arch/arm/dts/stm32mp131.dtsi
@@ -218,6 +218,8 @@
 
 		pwr_irq: pwr@50001010 {
 			compatible = "st,stm32mp1-pwr-irq";
+			wakeup-parent = <&exti>;
+			wakeup-gpios = <0>, <0>, <0>, <0>, <0>, <0>;
 			status = "disabled";
 		};
 
-- 
2.47.2

